src/vs/platform/terminal/common/capabilities/commandDetectionCapability.ts

```diff
@@ -348,6 +348,14 @@ export class CommandDetectionCapability implements ICommandDetectionCapability {
 		});
 	}
 
+	handleGenericCommand(properties: IGenericCommandProperties): void {
+		this.setIsCommandStorageDisabled();
```

Comment:
Since these sequences will be used along with regular shell integration sequences we will want to only apply this command storage disabled concept for this specific command. Because of this should we pass an extra property in to `handleCommandFinished` or something to say accept it even though `command` isn't set?